data/brands/amenity/parcel_locker.json

```diff
@@ -170,12 +170,12 @@
     {
       "displayName": "easybox",
       "id": "easybox-ba4d23",
-      "locationSet": {"include": ["ro"]},
+      "locationSet": {"include": ["ro", "hu"]},
       "tags": {
         "amenity": "parcel_locker",
         "brand": "easybox",
```

Comment:
Note that since this is in the brands folder, a `brand:wikidata` tag is needed for this to show up as an iD preset.

Reply:
Here: https://www.wikidata.org/wiki/Q114496224

Should I open a new PR?

Reply:
> Here: https://www.wikidata.org/wiki/Q114496224

Added it in https://github.com/osmlab/name-suggestion-index/commit/bbf233215d1c4c6f503b4ec48fd486bbead94696